include/CppUTest/CppUTestConfig.h

```diff
@@ -199,6 +199,27 @@
 #define CPPUTEST_CHAR_BIT 8
 #endif
 
+/* Support for "long long" type */
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
Ah.  No.  In the original January version of the patch I tried to be more clever and autodetect compiler support (there's still a few traces of that lingering in this version, such as `AC_TYPE_LONG_LONG_INT` that's not actually used other than to taunt people who read `config.h`).  But as you may recall that sometimes ended up with weird behaviour when eg. the C++ and C compilers detected differently.  So now it's just a simple manually-specified toggle.

Having said that, there **is** a basic auto-detection I could add back in, possibly -- just a few lines up you do (conditionally) include `limits.h`.  This does contain something that should theoretically tell us whether the compiler supports `long long`s (if `LLONG_MAX` is defined).

Although it's possible that some CRT libs define this even if the compiler doesn't support it, or that the application wants to build without them because they want to compile against an older C/C++ standard.  So for those reasons it might still be better to stick with a manual define.

(Though again, in the January PR I had it tristate -- you could define `CPPUTEST_USE_LONG_LONG` to force support on, `CPPUTEST_NO_LONG_LONG` to force support off, and neither to let it try to auto-detect.)